src/capture_RPiHQ.cpp

```diff
@@ -1369,8 +1331,7 @@ displayDebugText(debugText, 3);
 
 			if (myModeMeanSetting.mode_mean) {
```

Comment:
After the capture, other actions may happen (resize, stretch, ..) So I'm not sure whether the calculation should not be done after the sleep

Reply:
Looking ahead, it would be nice if RPiHQcalcMean() returned a pointer to the in-memory image, then we could use that to add an over lay, resize, stretch, etc., then save the final image.jpg file.  This is basically what the ZWO version does, but it doesn't have to re-read the image.jpg file.
So, I suggest moving the RPiHQcalcMean() a few lines up, to above the "// Check for night time".

**For this PR, I suggest we leave it as is.**